hack/tools/go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/metal3-io/ip-address-manager/hack/tools
 
-go 1.22
+go 1.22.0
```

Comment:
LGTM https://github.com/metal3-io/ip-address-manager/pull/562

it has all these hack/tools things done for you.